README.md

```diff
@@ -20,36 +20,12 @@ All the MiniApp files downloaded by the MiniApp iOS library are cached locally
 
```

Comment:
You also need to update this line in `.travis.yml` to point to `USERGUIDE.md`:
https://github.com/rakutentech/ios-miniapp/blob/29f696b2920d23af77da0a31ee0dfc5c9b976cc3/.travis.yml#L45

Reply:
Done. Updated